src/pages/ChairConsole.js

```diff
@@ -344,23 +344,11 @@ export const ChairConsole = hh(class ChairConsole extends Component {
                       isRendered: this.isAccessCollectEnabled(pendingCase),
                       className: twoColumnClass + ' cell-body f-center'
                     }, [
-                      button({
-                        id: pendingCase.frontEndId + '_btnCollect',
-                        name: 'btn_collectAccess',
-                        onClick: this.openAccessCollect(pendingCase.referenceId, pendingCase.electionId),
-                        className: 'cell-button cancel-color'
-                      }, ['Collect Votes'])
                     ]),
                     div({
                       isRendered: (pendingCase.alreadyVoted === true) && (pendingCase.electionStatus === 'Final'),
                       className: twoColumnClass + ' cell-body f-center'
                     }, [
-                      button({
-                        id: pendingCase.frontEndId + '_btnFinal',
-                        name: 'btn_finalAccess',
-                        onClick: this.openFinalAccessReview(pendingCase.referenceId, pendingCase.electionId, pendingCase.rpElectionId),
```

Comment:
The method, `openFinalAccessReview`, is no longer used so we can delete that.

Reply:
Looking into `openFinalAccessReview`, since that is no longer used, we also no longer need the page that it refers to, which is `FinalAccessReview`. That whole code path can be deleted.